docs/.vitepress/config.js

```diff
@@ -541,7 +541,15 @@ function sidebar() {
 					text: 'Creating Extensions',
 				},
 				{
-					text: 'Extension Types',
+					link: '/extensions/publishing-extensions',
+					text: 'Publishing Extensions',
+				},
+				{
+					link: '/extensions/installing-extensions',
+					text: 'Installing Extensions',
+				},
+				{
+					text: 'Developing Extensions',
```

Comment:
The content of each subpage is really focused on developing the particular extension - why I titled it "Developing Extensions" but I can understand how it could conflict with "Creating Extensions". Even at that, I'm not entirely convinced it should be called Extension Types though...

Reply:
I agree. Ask the team?

Reply:
Agree on the oddness!
Would it make sense to make the "Creating Extensions" the first sub-page of the "Developing Extensions" section?
At the moment, the order is also a bit confusing. If we continue with these names it should be more like this:
- Installing Extensions
- Creating Extensions
- Developing Extensions

Reply:
Thought about this and think it makes sense to have the "Creating Extensions" page as a stand alone the way it is instead of putting it as a sub page. Mainly because it contains subsections within it that we want users to easily access without it being hidden under "Developing Extensions". I'll make the structure for now:

- Installing Extensions
- Creating Extensions
- Developing Extensions